app/models/site_status.rb

```diff
@@ -19,6 +20,21 @@ class SiteStatus < ApplicationRecord
 
   after_initialize :set_defaults
   before_validation :set_vac_status
+  before_create :set_new_status
+
+  def set_new_status
+    if !course.site_statuses.empty? && !course.site_statuses.all?(&:status_new_status?)
+      self.status = 'R'
```

Comment:
```suggestion
      self.status = : running
```

Reply:
Just wondering that this change will affect the factory...

Reply:
@defong I'll have a look.